src/tape_drivers/linux/lin_tape/lin_tape_ibmtape.c

```diff
@@ -2231,6 +2231,7 @@ int lin_tape_ibmtape_logsense(void *device, const uint8_t page, const uint8_t su
 
 	if (rc != DEVICE_GOOD) {
 		lin_tape_ibmtape_process_errors(device, rc, msg, "logsense page", true);
+		return -1;
```

Comment:
I think you can return `rc` here directly.

It must have negative value on error or 0 on success. Because it is the return code of the `_sioc_stioc_command`. No positive value is stored even if `LEGSNS10` is successfully done.

Reply:
Thanks looks good to me.